.circleci/config.yml

```diff
@@ -204,6 +204,20 @@ jobs:
           path: /artifacts
           name: Artifact storage
 
+      - run:
+          command: |
+            /home/circleci/go/bin/ghr \
+            -draft \
+            -prerelease \
+            -r $GITHUB_REPOSITORY \
+            -u $GITHUB_USERNAME \
+            -t $GITHUB_TOKEN \
```

Comment:
I see these variables are not set in CircleCI on the Drud/ddev config. How can they be set in Circleci without exposing them to forked builds?

Reply:
The proposal in this PR is to:

1) Check public environment variables into Git and load them at the beginning of a job
2) Update Circle to make environment variables private and prevent them from being shared with forked builds